android-test/src/androidTest/java/okhttp/android/test/OkHttpTest.kt

```diff
@@ -17,102 +17,168 @@ package okhttp.android.test
 
 import android.os.Build
 import android.support.test.runner.AndroidJUnit4
-import okhttp3.Call
-import okhttp3.Connection
-import okhttp3.EventListener
-import okhttp3.OkHttpClient
-import okhttp3.Protocol
-import okhttp3.Request
-import okhttp3.TlsVersion
-import org.junit.After
+import okhttp3.*
```

Comment:
No star imports!

Reply:
Another one because of different defaults in Android Studio.